generators/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "Generators": {
+      "commandName": "Project",
+      "commandLineArgs": "-e allergies"
```

Comment:
Should this line be checked in? Isn't this only for testing?

Reply:
It is. I'll remove it.